app/models/avalon/rdf_vocab.rb

```diff
@@ -0,0 +1,65 @@
+# Copyright 2011-2018, The Trustees of Indiana University and Northwestern
```

Comment:
I think you can remove this whole file since you're using ::RDF::URI.new to declare the predicate uri in the model mixin.